site/docs/resources/faq.md

```diff
@@ -13,45 +13,45 @@ It reached the first stable version a few weeks later.
 
 ## How is grammY developed?
 
-grammY is completely free and open-source software, developed by a team of volunteers.
+grammY is a completely free and open-source software, developed by a team of volunteers.
```

Comment:
```suggestion
grammY is completely free and open-source software, developed by a team of volunteers.
```

Reply:
I think there is one and only grammY, so this suggestion is invalid.